app/controllers/audits_controller.rb

```diff
@@ -69,7 +69,7 @@ def create
     if @audit.save
       save_audit_status_and_redirect(params)
     else
-      flash[:error] = @audit.errors.collect { |model, message| "#{model}: " + message }
+      flash[:error] = "<ul><li>" + @audit.errors.collect { |model, message| "#{model}: " + message }.join("</li><li>") + "</li></ul>"
```

Comment:
Your PR description says...

>The `<ul><li>` tags are wrapped in the controller layer instead of in the model layer. This isn't quite as ideal as getting into the view layer, but it's a step in the right direction

I see the addition here, but where is it getting removed from the model layer1?

Reply:
It looks like that's still lingering in the `app/models/errors.rb` class. Let me go see if I can extract that -- you're correct, this was a rather important part!

Reply:
Ah, nevermind -- we're using `sanitize` with whitelisted HTML tags. It's all good. :+1: